### PR TITLE
New values.yaml reference page

### PR DESCRIPTION
**Pull request by StackScribe**
This minimal values.yaml reference page.  In lieu of showing the file "Synopsis", this includes a link to the spec in the keptn/keptn repository.  This is a temporary measure until we determine what we can do with the new documentation engine.  Meanwhile, this spares users the effort of slogging through the source repo to find this information and we can add appropriate information as it arises plus reference it from other docs.

This draft includes a "Differences between versions" section that preserves information about changes made in 0.17.x to this file.

### CODE REVIEW

**Review by thschue — COMMENTED** [review]
lgtm

**Review by thschue — APPROVED** [review]
lgtm

File: content/docs/0.18.x/reference/files/values/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: values (Keptn helm-service)
+description: Configure the Keptn Helm chart
+weight: 915
+---
+
+The `values.yaml` configuration file defines the configuration used
+when installing Keptn with a Helm chart; see
+[Installing Keptn using the Helm chart](../../../../install/helm-install).
+`Values` is a built-in object of Helm templates; see the Helm documentation at
+[Values Files(https://helm.sh/docs/chart_template_guide/values_files/).
```
Comment (mowies) [inline content/docs/0.18.x/reference/files/values/index.md:11]:
```suggestion
[Values Files](https://helm.sh/docs/chart_template_guide/values_files/).
```

File: content/docs/0.18.x/reference/files/values/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Spec
+
+See [values.yaml](https://github.com/keptn/keptn/blob/master/installer/manifests/keptn/values.yaml)
+in the Keptn GitHub repository for the default Keptn file.
```
Comment (mowies) [inline content/docs/0.18.x/reference/files/values/index.md:16]:
```suggestion
in the Keptn GitHub repository for Keptn's default Helm values file.
```

File: content/docs/0.18.x/reference/files/values/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## Usage
+
+The recommended practice is the declarative approach:
+you create your own Keptn values file that includes your customizations
+then use your customized values file to install Keptn.
+Alternatively, you can also set Helm values with `--set` during helm commands;
+this is the imparative approach.
```
Comment (mowies) [inline content/docs/0.18.x/reference/files/values/index.md:24]:
```suggestion
this is the imperative approach.
```

File: content/docs/0.18.x/reference/files/values/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+The --set flags take precendence over whatever you have configured in your *values.yaml* file
+so can be used to temporarily override a value when necessary.
+
+## Files
+
+* Default [values.yaml](https://github.com/keptn/keptn/blob/master/helm-service/chart/values.yaml#L10) file:
```
Comment (mowies) [inline content/docs/0.18.x/reference/files/values/index.md:30]:
```suggestion
* Default [values.yaml](https://github.com/keptn/keptn/blob/master/installer/manifests/keptn/values.yaml) file: 
```

File: content/docs/0.18.x/reference/files/values/index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/0.18.x/reference/files/values/index.md:11]: done

File: content/docs/0.18.x/reference/files/values/index.md (hunk 3 above)
Comment (StackScribe) [reply content/docs/0.18.x/reference/files/values/index.md:24]: done

File: content/docs/0.18.x/reference/files/values/index.md (hunk 4 above)
Comment (StackScribe) [reply content/docs/0.18.x/reference/files/values/index.md:30]: done

File: content/docs/0.18.x/reference/files/values/index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/0.18.x/reference/files/values/index.md:16]: done

**Pull request merged**